src/applications/vaos/tests/reducers/appointments.unit.spec.js

```diff
@@ -86,6 +95,15 @@ describe('VAOS reducer: appointments', () => {
           },
         ],
         [
+          // CC appointment scheduled less than 395 days into the future should show
+          {
+            appointmentTime: moment()
+              .add(394, 'days')
+              .format(),
```

Comment:
I think you need to pass a format string to `format()` so that the result looks like `05/29/2099 05:30:00`. I believe it's the format string in the second parameter here: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/utils/appointment.js#L74

The VA appointments use the standard ISO format, so just doing `format()` gets the right output. For some reason, Community Care appointments have a different, custom format.

Reply:
@jbalboni, thanks for pointing that out. Would changing ".format()" to ".format('MM/DD/YYYY HH:mm:ss')" do the trick?

Reply:
@jenniemc Yep, that should work.

Reply:
@jbalboni, replacing with  ".format('MM/DD/YYYY HH:mm:ss')" had failed the unit test

Reply:
I don't think the format is what's wrong. As far as I can tell, this is what's going on:

1. The test has 5 items across the three arrays, and there's an expect statement checking that there are 5 appointments after running the code
2. You added two more items to the arrays, bringing the total up to 7
3. You added an additional check to the filter function that limits to 395 days. That will remove the two 2099 appointments, bringing your total back down to 5 appointments
4. You changed the expect statement to expect 4 appointments. That seems incorrect, given your other changes?

Reply:
@jbalboni, I had originally ruled out the last array, ` [{ optionDate1: '05/29/2099' }],` as part of the items that won't displays because it is not a confirmed appt.  I understand now this unit test checks for confirmed future appts and pending appts. Thanks for the correct count. I'll make clarification to the comments.